packages/plugins/documentation/server/src/services/helpers/build-api-endpoint-path.ts

```diff
@@ -18,15 +18,24 @@ import type { Api, ApiInfo } from '../../types';
  * @param {string} routePath - The route's path property
  * @returns {string}
  */
-const parsePathWithVariables = (routePath: string) => {
-  return pathToRegexp
-    .parse(routePath)
+const parsePathWithVariables = (routePath: string): string => {
+  const { tokens } = pathToRegexp.parse(routePath);
+
+  return tokens
     .map((token) => {
-      if (_.isObject(token)) {
-        return `${token.prefix}{${token.name}}`;
+      switch (token.type) {
```

Comment:
To explain this, in path-to-regexp v8, `prefix` no longer exists and its functionality has been replaced with `'text'` type tokens.

`Token` is now a standardized object with one of the following types

https://www.npmjs.com/package/path-to-regexp/v/8.0.0

Reply:
am I right that when you bump the documentation plugin version and run the server I should see new docs generated in?

1 - examples/getstarted/src/extensions/documentation/documentation

2 - On content types e.g:
examples/getstarted/src/api/category/documentation

Because I dont see number 2 working.

Reply:
on 2, I don't see that happening even on `develop`. Did v4 used to do that? Or am I missing something?

Reply:
Yes v4 is doing that. This must be a bug in v5 too then?

Reply:
In which case I'm happy to merge this PR and log that bug elsewhere. wdyt?